src/app/admin/components/editing-user-data/editing-user-data.component.html

```diff
@@ -2,39 +2,50 @@
   <form [formGroup]="form" class="d-flex flex-column">
     <mat-form-field class="form-input">
       <mat-label>Username:</mat-label>
-      <input matInput type="text" formControlName="username">
-      <mat-error *ngIf=" userValue?.invalid">Field cannot be empty</mat-error>
+      <input matInput type="text" formControlName="username" />
+      <mat-error *ngIf="userValue?.invalid">Field cannot be empty</mat-error>
     </mat-form-field>
 
     <mat-form-field class="form-input">
       <mat-label>Role:</mat-label>
       <mat-select formControlName="role">
         <mat-option *ngFor="let roleType of roles" [value]="roleType">
-          {{roleType}}
+          {{ roleType }}
         </mat-option>
         <mat-error *ngIf="roleValue?.invalid">Please choose an role</mat-error>
       </mat-select>
     </mat-form-field>
 
     <mat-form-field class="form-input">
       <mat-label>Password:</mat-label>
-      <input matInput type="password" formControlName="password">
-      <mat-error *ngIf="passwordValue?.errors?.['required']">Field can't be empty</mat-error>
-      <mat-error *ngIf="passwordValue?.errors?.['minlength']">Length must be at least 10
-        characters. You have {{passwordValue?.value.length}} characters.</mat-error>
+      <input matInput type="password" formControlName="password" />
+      <mat-error *ngIf="passwordValue?.errors?.['required']"
+        >Field can't be empty</mat-error
+      >
+      <mat-error *ngIf="passwordValue?.errors?.['minlength']"
+        >Length must be at least 10 characters. You have
+        {{ passwordValue?.value.length }} characters.</mat-error
```

Comment:
Could you setup prettier that it format like that 

 ```
<mat-error
        *ngIf="!passwordValue?.errors?.['minlength'] && compareValidator"
        >
    Passwords do not match
</mat-error>
```

Reply:
not quite but close
new commit added